homeassistant/components/ihc/sensor.py

```diff
@@ -64,12 +71,12 @@ def device_class(self):
         )
 
     @property
-    def native_value(self):
+    def native_value(self) -> StateType | date | datetime | Decimal:
```

Comment:
This is type hint improvement - unrelated to config-flow. Please move to preliminary/follow-up PR.